contracts/MemeArbitrum.sol

```diff
@@ -103,7 +100,10 @@ contract MemeArbitrum is MemeFactory {
             payable(address(this)), false);
 
         // Perform swap
-        return IBalancer(balancerVault).swap(singleSwap, fundManagement, 0, block.timestamp);
+        uint256 olasAmount = IBalancer(balancerVault).swap(singleSwap, fundManagement, 0, block.timestamp);
+
+        // Apply slippage protection
```

Comment:
Slippage protection must be run at the end. Otherwise, the swap might lead to massive slippage and go undetected...

Reply:
@77ph please provide input.